pkg/operator/ceph/object/controller_test.go

```diff
@@ -745,6 +791,68 @@ func TestCephObjectStoreControllerMultisite(t *testing.T) {
 		assert.True(t, dependentsChecked)
 		assert.True(t, calledCommitConfigChanges)
 	})
+
+	t.Run("create an external object store", func(t *testing.T) {
+		objects := []runtime.Object{
+			objectZone,
+			objectZoneGroup,
+			objectRealm,
+			cephCluster,
```

Comment:
Do objectZone, objectZoneGroup, objectRealm applicable to external rgw cluster?

Reply:
Removed.